packages/ckeditor5-bookmark/docs/api/bookmark.md

```diff
@@ -0,0 +1,36 @@
+---
+category: api-reference
+---
+
+# CKEditor&nbsp;5 bookmark feature
+
+[![npm version](https://badge.fury.io/js/%40ckeditor%2Fckeditor5-block-quote.svg)](https://www.npmjs.com/package/@ckeditor/ckeditor5-bookmark)
+
+This package implements bookmark support for CKEditor&nbsp;5.
+
+## Demo
+
+Check out the {@link features/bookmarks#demo demo in the bookmarks feature guide}.
+
+## Documentation
+
+See the {@link features/bookmarks bookmarks feature guide} and the {@link module:block-quote/blockquote~BlockQuote} plugin documentation.
```

Comment:
@godai78  Is the plugin documentation link correct?

Reply:
Yes, it was updated recently.